internal/provider/app_owner_resource_sdk.go

```diff
@@ -0,0 +1,55 @@
+package provider
+
+import (
+	"conductorone/internal/sdk/pkg/models/shared"
+
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func (r *AppOwnerResourceModel) ToCreateSDKType() *shared.SetAppOwnersRequest {
+	var userIds []string = nil
+	for _, userIdsItem := range r.UserIds {
+		userIds = append(userIds, userIdsItem.ValueString())
+	}
+	out := shared.SetAppOwnersRequest{
+		UserIds: userIds,
+	}
+	return &out
+}
+
+func (r *AppOwnerResourceModel) RefreshFromCreateResponse(resp *shared.SetAppOwnersResponse) {
```

Comment:
Unimplemented? Should this just be deleted

Reply:
I think maybe lets just leave it since it is generated like this.